doc/10-icinga-template-library.md

```diff
@@ -3278,6 +3278,132 @@ printer_health_oids		| **Optional.** A list of oids which are downloaded and wri
 printer_health_offline		| **Optional.** The maximum number of seconds since the last update of cache file before it is considered too old.
 printer_health_multiline	| **Optional.** Multiline output
 
+#### Thola <a id="plugin-contrib-command-thola"></a>
+
+The [Thola](https://thola.io) plugin
+is a tool for monitoring network devices, that mainly uses SNMP.
+To monitor devices the subcommand `thola check` can be used.
+
+`Templates:`
+
+If you specify the Thola API address, snmp community and version in the following templates, then they will be 
+automatically used for each Thola subcommand.
+
+**generic-thola-check-command:**
+
+Name                    | Description
+------------------------|----------------------------------------
+thola_api_address       | Address of the Thola API to connect to
+
+**generic-thola-device-check-command:**
+
+Name                                 | Description
+-------------------------------------|------------------------------
+thola_device_address                 | IP address of the device
+thola_device_snmp_community          | SNMP community of the device
+thola_device_snmp_protocol           | SNMP version to use
+
+`Commands:`
+
+The `thola check` command has eight different subcommands:
+
+A documentation of these commands can be found by running `thola check -h`
+or in the plugin's API [swaggerhub documentation](https://app.swaggerhub.com/apis-docs/thola/thola/).
+To run these commands you need a server that is running the Thola API.
+If you don't know how to do this, you can also have a look at the plugin's
+[documentation](https://docs.thola.io). Also you have to
+put the Thola-client binary into the *PluginContribDir*.
+
+#### thola-check-cpu-load <a id="plugin-contrib-command-thola-check-cpu-load"></a>
+`thola check cpu-load` checks the cpu-load of a network device.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                                   | Description
+---------------------------------------|--------------------------------------------------------
+thola_cpu_load_critical                | **Optional.** Sets critical threshold for the cpu load
+thola_cpu_load_warning                 | **Optional.** Sets warning threshold for the cpu load
+
+#### thola-check-interface-metrics <a id="plugin-contrib-command-thola-check-interface-metrics"></a>
+`thola check interface-metrics` checks the interface-metrics of a network device.
+
+There are no custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters).
```

Comment:
At the first sight e.g. this looks like the command actually doesn’t take anything. But it **does** as it imports the template. Therefore...